package.json

```diff
@@ -1,97 +1,106 @@
-    "presample-app": "npm i dist/npm/clarity-ui; npm i dist/npm/clarity-icons; npm i dist/npm/clarity-angular;",
-    "sample-app": "gulp sample",
-    "preks:publish": "gulp npm:prepare --version $npm_package_version",
-    "ks:publish": "cd src/ks-app; ng build --preserve-symlinks --prod; cd ../../dist/ks-app; cat index.html > 404.html; surge; cd ../../",
-    "cssTest": "./docker-ctrl/cdc.js -t",
-    "cssUpdate": "./docker-ctrl/cdc.js -u"
-  }
+    "name": "clarity-all",
+    "version": "0.10.6",
+    "private": true,
+    "scripts": {
+        "start": "webpack-dev-server --port=4200 --host 0.0.0.0",
+        "clean": "del ./dist",
+        "build": "npm-run-all clean build:angular webpack build:icons",
+        "pretest": "webpack",
+        "test": "karma start ./karma.conf.js",
+        "test:travis": "npm-run-all clang:check tslint:check test",
+        "clang:check": "./scripts/clang-check.sh",
+        "clang:format": "node_modules/.bin/clang-format -style=file -i --glob=src/**/*.ts",
+        "prebuild:angular": "cp ./npm/clarity-angular/package.json ./src/clarity-angular/; replace '@VERSION' $npm_package_version ./src/clarity-angular/package.json",
+        "build:angular": "ng-packagr -p ng-package.json",
+        "postbuild:angular": "cd dist/clarity-angular; npm pack;",
+        "build:icons": "webpack --config webpack.icons.config.js",
+        "postbuild:icons": "node ./scripts/clarity-icons-svg.js ",
+        "npm:publish": "npm publish dist/clarity-ui; npm publish dist/clarity-angular/clarity-angular-$npm_package_version.tgz; npm publish dist/clarity-icons",
+        "cssTest": "./scripts/docker-cdc.js -t",
+        "cssUpdate": "./scripts/docker-cdc.js -u",
+        "replace": "replace",
+        "tslint:check": "node_modules/.bin/tslint -c tslint.json \"src/**/*.ts\" -e \"**/node_modules/**\"",
+        "tslint:fix": "node_modules/.bin/tslint -c tslint.json --fix \"src/**/*.ts\" -e \"**/node_modules/**\"",
+        "webpack": "webpack",
+        "preks:publish": "build",
+        "ks:publish": "cd src/ks-app; ng build --preserve-symlinks --prod; cd ../../dist/ks-app; cat index.html > 404.html; surge; cd ../../"
+    },
+    "dependencies": {
+        "@angular/animations": "^4.3.0",
+        "@angular/common": "^4.3.0",
+        "@angular/compiler": "^4.3.0",
+        "@angular/compiler-cli": "^4.3.0",
+        "@angular/core": "^4.3.0",
+        "@angular/forms": "^4.3.0",
+        "@angular/platform-browser": "^4.3.0",
+        "@angular/platform-browser-dynamic": "^4.3.0",
+        "@angular/router": "^4.3.0",
+        "@webcomponents/custom-elements": "^1.0.0",
+        "bootstrap": "4.0.0-alpha.5",
+        "core-js": "^2.4.1",
+        "font-awesome": "^4.5.0",
+        "prismjs": "^1.6.0",
+        "rxjs": "^5.4.2",
+        "web-animations-js": "^2.3.1",
+        "zone.js": "^0.8.14"
+    },
+    "devDependencies": {
+        "@angular/cli": "1.4.0",
```

Comment:
Do we have a good reason to lock this to version 1.4.0? If not, my preference is for `^1.4.0`

Reply:
I think the cli is sometimes a bit volatile with breaking changes so I wanted to lock it. I can change it to `^1.4.0` and we monitor closely as we go forward.

Reply:
Can do `~1.4.0` to keep it in 1.4 minor release.

Reply:
Updated to `~1.4.0`.

Reply:
My vote goes to pin exact version of the cli and ng-packagr as start. I've seen even patch version of the cli update sometimes gets into trouble. Updates of ng-packagr seem pretty often nowadays.

How about duplicated webpack plugins and loaders in the cli and your package.json? I am wondering which version will be picked at build if there are mismatched. Just for thoughts from here.